doc/md/getting-started.md

```diff
@@ -124,9 +138,12 @@ ent
 
 ## Create Your First Entity
 
-To get started, create a new `ent.Client`. For this example, we will use SQLite3.
+To get started, create a new `ent.Client`. For this example, we will use SQLite3.  
+First create an executable package `main`, in this example we will place it under `<project>/start/start.go`
```

Comment:
Need some rephrasing work here. 

cc @rotemtam

Reply:
I'll remove it since the "title" directive makes it clearer